src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/Helper.groovy

```diff
@@ -0,0 +1,21 @@
+package org.openkilda.functionaltests.helpers
+
+import com.github.javafaker.Faker
+
+import java.text.SimpleDateFormat
+
+class Helper {
+
+    static def random = new Random()
+    static def faker = new Faker()
+
+    static String generateDescription(String flowType) {
```

Comment:
Each flow-specific helper has generateDescription() and generateFlowId() methods, it seems like we need to move these methods to the separate class and reuse for different flow payload creation.
WDYT: As for the param 'flowType', we can add enum for the main flow types (ha-, y, regular).